### ISSUE TITLE
Update task table CI job infinite loop

### ISSUE THREAD

**Issue by isaac-chung** [embeddings-benchmark/mteb#540]
Looks like this CI job keeps triggering. Only the languages are changing order. We might need to sort the languages in the create task table script.

CC @KennethEnevoldsen

**Comment by isaac-chung** [comment]
Merged, let's observe for a few more commits to main.

**Comment by imenelydiaker** [comment]
The CI is still failing unexpectedly, see PR #527 and #539, I had to re-run jobs for #527 for them to pass. Any idea on why this is happening?

**Comment by KennethEnevoldsen** [comment]
@imenelydiaker I don't believe it is due to this issue but have created another issue for it #544

**Comment by isaac-chung** [comment]
This issue is looking at why we see so many "Update tasks table" [commits to main](https://github.com/embeddings-benchmark/mteb/commits/main/) (see screenshot below).

![Screenshot 2024-04-24 at 11 47 41 AM](https://github.com/embeddings-benchmark/mteb/assets/48971969/41bc12e6-a7be-4439-96b0-792b9d516365)

**Comment by isaac-chung** [comment]
My 1st PR does not seem to have fixed the issue. Languages are still shuffling in subsequent commits by the bot. Perhaps using an ordered dict or converting to a sorted list may help.

**Comment by KennethEnevoldsen** [comment]
^converting it to a list seems like a good approach

**Comment by KennethEnevoldsen** [comment]
Have merged this into main (will monitor to see how it goes)

**Comment by isaac-chung** [comment]
![image](https://github.com/embeddings-benchmark/mteb/assets/48971969/b42cd10f-87c6-4063-9ff9-b7debb3e10e6)

**Comment by isaac-chung** [comment]
This [commit](https://github.com/embeddings-benchmark/mteb/commit/6faae8658f47d5bebc79a665529740b472519856) was the true fix! Thanks @KennethEnevoldsen . Marking this as closed.
